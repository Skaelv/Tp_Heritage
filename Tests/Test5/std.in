C Objet 12 435 823
R Objet 12 12 17 18
EXIT
