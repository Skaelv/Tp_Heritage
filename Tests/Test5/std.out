 # Entrez une commande
OK
# New Object : Objet
 # Entrez une commande
ERR
 #Invalid Parameters
 # Entrez une commande

